api-reference/docstubs/api/microsoft.fast.pi.viewgenerator.v2.profile.models-profile-post-websites.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Create websites"
+description: "Create a new websites object."
+author: "**TODO: Provide Github Name**"
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# Create websites
+
+Namespace: Microsoft.Fast.PI.ViewGenerator.V2.Profile.Models
+
+Create a new websites object.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Provide applicable permissions.**|
+|Delegated (personal Microsoft account)|**TODO: Provide applicable permissions.**|
+|Application|**TODO: Provide applicable permissions.**|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+POST /user/{userId}/profile/websites
+```
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required|
+|Content-Type|application/json. Required|
+
+## Request body
+In the request body, supply a JSON representation for the [personWebsite](../resources/microsoft.fast.pi.viewgenerator.v2.profile.models-personwebsite.md) object.
+
+The following table shows the properties that are required when you create the [personWebsite](../resources/microsoft.fast.pi.viewgenerator.v2.profile.models-personwebsite.md).
+
+|Property|Type|Description|
+|:---|:---|:---|
+|id|String|**TODO: Add Description** Inherited from [itemFacet](../resources/microsoft.fast.pi.viewgenerator.v2.profile.models-itemfacet.md)|
```

Comment:
Should the inheritance statement be on a separate line?  You'll need to add a line break.

Reply:
+1 for the line break.